3_Implementation/documentation/html/search/functions_3.js

```diff
@@ -0,0 +1,6 @@
+var searchData=
+[
+  ['setup_832',['setUp',['../_t_i_c-_t_a_c-_t_o_e__unittest_8c.html#a63326a1b70b6e3f070e51687798e02de',1,'setUp():&#160;TIC-TAC-TOE_unittest.c'],['../unity_8h.html#a95c834d6178047ce9e1bce7cbfea2836',1,'setUp(void):&#160;TIC-TAC-TOE_unittest.c']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["setup_832",['setUp',['../_t_i_c-_t_a_c-_t_o_e__unittest_8c.html#a63326a1b70b6e3f070e51687798e02de',1,'setUp():&#160;TIC-TAC-TOE_unittest.c'],['../unity_8h.html#a95c834d6178047ce9e1bce7cbfea2836',1,'setUp(void):&#160;TIC-TAC-TOE_unittest.c']]],
```